multidim-interop/README.md

```diff
@@ -22,14 +22,39 @@ environment variables. The current parameters are:
 | muxer                | The muxer to use                                             | no, except when transport is one of quic, quic-v1, webtransport |
 | security             | The security channel to use                                  | no, except when transport is one of quic, quic-v1, webtransport |
 | is_dialer            | Should you dial or listen                                    | no                                                              |
-| ip                   | IP address to bind the listener to                           | yes, default to "0.0.0.0"                                       |
+| ip                   | IP address to bind the listener to                           | yes, except rust - default to "0.0.0.0"                         |
```

Comment:
Sounds like this should be an issue on the rust impl rather than having to document how each impl doesn't adhere to this spec.